### ISSUE TITLE
Rif1 Knockdown Hi-C

### ISSUE THREAD

**Issue by kyleklein** [PPSCC/ppsc#5]
Hi Steve and Li. This GitHub issue is to discuss analysis of our new Rif1 knockout Hi-C data. The first thing we want to know is whether there are any effects of Rif1 knockout on either A/B compartments or TAD border strenghts. Please let us know if we can answer any questions.

**Comment by inyourgenes** [comment]
Got it!! Thanks Steve! Kyle and Vishnu will look at the heat maps Li sent and get back. In the meantime, we are interested in 1) global border strength and compartmentalization (WT vs. Rif1 KD) and after that 2) border strength and compartmentalization at regions of the genome affected for replication timing, which Kyle and Vishnu and get you guys. If it is a complicated analysis, we could focus on #2, I just thought it would be easier to run the whole genome first.

**Comment by vcorces** [comment]
I think in order to get that information we would need at least 200 million processed reads for each sample. Right now we only have 6 million. The idea was that this is all we needed to see the compartments

**Comment by inyourgenes** [comment]
there may have been some confusion. The experiment that Korey is preparing (cell cycle time points after differentiation) only needs compartment information, but for Rif1 a big question is whether it affects compartments or TAD boundaries (either, both or neither). If all we can see is compartments from this first experiment, then OK - we will sequence deeper for the repeat. We can do it here at FSU faster and cheaper. We will look at the data but does Li have an opinion on whether compartments can be seen?

**Comment by vcorces** [comment]
ok, I didn't realize this. If you want the libraries, we need to ask HA for them.

**Comment by vcorces** [comment]
by the way, before you decide to sequence these libraries again, keep in mind that, for whatever reason, Li was able to map only around 40% of the reads. I don't know if he looked into why in detail, if it is a matter of the sequence quality of something wrong with the library. Li, did you check the quality of the fastq files?

**Comment by inyourgenes** [comment]
I know that Kyle is making replicates.


> On Oct 1, 2015, at 8:58 AM, vcorces <notifications@github.com> wrote:
> 
> by the way, before you decide to sequence these libraries again, keep in mind that, for whatever reason, Li was able to map only around 40% of the reads. I don't know if he looked into why in detail, if it is a matter of the sequence quality of something wrong with the library. Li, did you check the quality of the fastq files?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/PPSCC/ppsc/issues/5#issuecomment-144771388>.
> 

Sincerely,

Dave

***************************************************** 
Dr. David M. Gilbert 
J. Herbert Taylor Distinguished Professor of Molecular Biology
Florida State University Distinguished Research Professor
Department of Biological Science 
319 Stadium Drive 
Florida State University 
Tallahassee, FL 32306-4295 
Phone: 850-645-7583 
Fax: 850-645-8447 
Email: gilbert@bio.fsu.edu <mailto:gilbert@bio.fsu.edu> 
Lab page: http://gilbertlab.bio.fsu.edu/ <http://gilbertlab.bio.fsu.edu/> 
ReplicationDomain Database: http://www.replicationdomain.org <http://www.replicationdomain.org/>
 *****************************************************

**Comment by kyleklein** [comment]
I will made replicates, should I fix more cells this time so we could potentially get more reads? Maybe 10 million instead of 5 million?

**Comment by inyourgenes** [comment]
Yes - more cells makes obtaining more reads more meaningful of course as there will be more interaction events.

I’ve been assuming that you run a western or Q-PCR at the time of collection to determine the extent of knockdown in the collected cell population - yes??

> On Oct 1, 2015, at 9:42 AM, kyleklein <notifications@github.com> wrote:
> 
> I will made replicates, should I fix more cells this time so we could potentially get more reads? Maybe 10 million instead of 5 million?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/PPSCC/ppsc/issues/5#issuecomment-144782715>.
> 

Sincerely,

Dave

***************************************************** 
Dr. David M. Gilbert 
J. Herbert Taylor Distinguished Professor of Molecular Biology
Florida State University Distinguished Research Professor
Department of Biological Science 
319 Stadium Drive 
Florida State University 
Tallahassee, FL 32306-4295 
Phone: 850-645-7583 
Fax: 850-645-8447 
Email: gilbert@bio.fsu.edu <mailto:gilbert@bio.fsu.edu> 
Lab page: http://gilbertlab.bio.fsu.edu/ <http://gilbertlab.bio.fsu.edu/> 
ReplicationDomain Database: http://www.replicationdomain.org <http://www.replicationdomain.org/>
 *****************************************************

**Comment by kyleklein** [comment]
I hadn't taken any cells for Western from the pool that went to Hi-C last time because it has stayed knocked down in all the Repli-seq analysis I've done, but I will do it this time just in case the levels have changed.

**Comment by vcorces** [comment]
I'm confused as to how I should reply to a specific comments. With the smoke signals, you just use a different type of smoke. Anyway, 5 million cells is what Rao et al used for all their libraries using the in situ procedure. The % of reads that map is somewhere 40-60 %, so it's not going to get much better than this. A second problem we've had with mammalian libraries is the high % of self ligation. We use 5 million Drosophila cells for Drosophila libraries and we don't have this problem. Xiaowen claims, although I don't understand the basis for it, that using fewer cells reduces the self ligation problem. She made some libraries with 1 million cells and the self ligation was way lower. So, I would suggest that you don't make more than 5 million cells.
The low number of reads we got is because I misunderstood what you guys wanted to see in the results. Back in August when we were trying to decide on whether to sequence just a few reads or to sequence the libraries in one lane each, Dave decided to be cheap and said: "OK - so we didn’t hear from Kyle on this so I’ll make the call. I don’t know how much we are talking but I’m assuming not that much (guessing a few hundred bucks?), so I would do both - go ahead and tell them to sequence the existing samples so we get an idea of compartments, and then remake the libraries so we have a good replicate later to work with". So that's where we are now. We sequenced a few million reads to get an idea of compartments and Kyle is making the replicates. I would suggest that you send us the cells in two tubes each, one with 1 million and one with 5 million. We can make the four libraries, sequence a trace and decide which ones to sequence more deeply.

**Comment by inyourgenes** [comment]
Thanks!!!  Your smoke signals are very clear. You should consider this like we were all in a room chatting, except that you use email and everything we say is recorded so we can re-visit more easily than trying to remember who said what when with what subject line in the email string. 

Vishnu may also have something to say about this. I could also talk with Job and the ENCODE guys about this self-ligation thing (we have a 3D bi-weekly call to discuss these issues, but I am always ignorant on the technical and computational details so I just listen and check email until something interesting happens, like what is the next figure of the manuscript).




> On Oct 1, 2015, at 10:18 AM, vcorces <notifications@github.com> wrote:
> 
> I'm confused as to how I should reply to a specific comments. With the smoke signals, you just use a different type of smoke. Anyway, 5 million cells is what Rao et al used for all their libraries using the in situ procedure. The % of reads that map is somewhere 40-60 %, so it's not going to get much better than this. A second problem we've had with mammalian libraries is the high % of self ligation. We use 5 million Drosophila cells for Drosophila libraries and we don't have this problem. Xiaowen claims, although I don't understand the basis for it, that using fewer cells reduces the self ligation problem. She made some libraries with 1 million cells and the self ligation was way lower. So, I would suggest that you don't make more than 5 million cells.
> The low number of reads we got is because I misunderstood what you guys wanted to see in the results. Back in August when we were trying to decide on whether to sequence just a few reads or to sequence the libraries in one lane each, Dave decided to be cheap and said: "OK - so we didn’t hear from Kyle on this so I’ll make the call. I don’t know how much we are talking but I’m assuming not that much (guessing a few hundred bucks?), so I would do both - go ahead and tell them to sequence the existing samples so we get an idea of compartments, and then remake the libraries so we have a good replicate later to work with". So that's where we are now. We sequenced a few million reads to get an idea of compartments and Kyle is making the replicates. I would suggest that you send us the cells in two tubes each, one with 1 million and one with 5 million. We can make the four libraries, sequence a trace and decide which ones to sequence more deeply.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/PPSCC/ppsc/issues/5#issuecomment-144790985>.
> 

Sincerely,

Dave

***************************************************** 
Dr. David M. Gilbert 
J. Herbert Taylor Distinguished Professor of Molecular Biology
Florida State University Distinguished Research Professor
Department of Biological Science 
319 Stadium Drive 
Florida State University 
Tallahassee, FL 32306-4295 
Phone: 850-645-7583 
Fax: 850-645-8447 
Email: gilbert@bio.fsu.edu <mailto:gilbert@bio.fsu.edu> 
Lab page: http://gilbertlab.bio.fsu.edu/ <http://gilbertlab.bio.fsu.edu/> 
ReplicationDomain Database: http://www.replicationdomain.org <http://www.replicationdomain.org/>
 *****************************************************
